common/lib/xmodule/xmodule/js/src/sequence/display.js

```diff
@@ -78,10 +77,21 @@
 
         Sequence.prototype.updatePageTitle = function() {
             // update the page title to include the current section
-            var positionLink = this.link_for(this.position);
+            var positionLink = this.link_for(this.position),
+                currentSectionTitle;
```

Comment:
I think you have to edit the CoffeeScript version of this (`display.coffee`) and not the generated JS. @bjacobel: Is that right? Sequence Module is still running on Coffee, right?